src/injected/visualization/root-container-creator.ts

```diff
@@ -9,6 +9,6 @@ export class RootContainerCreator {
         this.htmlElementUtils.deleteAllElements(`#${id}`);
         const root = document.createElement('div');
         root.id = id;
-        this.htmlElementUtils.querySelector('body').appendChild(root);
+        this.htmlElementUtils.querySelector('html').appendChild(root);
```

Comment:
Does adding a `<div>` element directly to the `<html>` element "just work"??? I'm curious about how chrome and edge handle this. Care to elaborate on your findings?

Reply:
Yes, since the html element is just another element. Since we don't really need to be on the body anyways, this isn't an issue.

Reply:
but is it allowed as per standards though? the browsers may be doing lots of things to suppress errors in the page. did u check the valid html children tags?

Reply:
i tested the validity here - https://validator.w3.org/nu/#textarea

with this content:
```
<!DOCTYPE html>
<html lang="en">
<head>
<title>title 1</title>
</head>

<body>
hello
</body>

<div>hi
</div>

</html>
```

Reply:
Posting the result of that validation:

![image](https://user-images.githubusercontent.com/2837582/77778741-b498fe80-700e-11ea-8af1-7e0e23e6d5e0.png)

Error: Stray start tag div.
From line 11, column 1; to line 11, column 5
↩</body>↩↩<div>hi↩</d